### ISSUE TITLE
How can i make i map like yours

### ISSUE THREAD

**Issue by TsengelBair** [kno20205/pathpal-indoor-navigation#3]
Hello, I'm doing a graduation project (indoor navigation on cruise lainer).
Could you explain me, how can i make paths (edges) like yours? 
The main difficulty lies in the edges, they must be broken so that the route does not go in a straight line, now i've got something like this
![hmROnhsBwxA](https://github.com/kno20205/indoor-wayfinder/assets/129286264/816d16fa-96d7-4aff-9d15-c4bec3318ffc)

May be you have some guide, how to make a map in Adobe and implement it into react?
Thank you)

**Comment by KnotzerIO** [comment]
I tried using Adobe Illustrator to create maps, but it wasn't very good for transferring the map to a React app. I drew circles for the map's points and connected them with lines just for visual effect. To use the map in my app, I wrote a script to change the circles' coordinates from the SVG file into JSON format. I manually defined the connections between points afterward. I planned to add a custom map editor to make this easier, but because of other projects, this won't happen soon.
